pepys_admin/admin_cli.py

```diff
@@ -1,99 +1,20 @@
-import argparse
 import cmd
-import datetime
 import os
+from datetime import datetime
 
 from iterfzf import iterfzf
 
-from config import DB_HOST, DB_NAME, DB_PASSWORD, DB_PORT, DB_TYPE, DB_USERNAME
-from pepys_import.core.store.data_store import DataStore
+from pepys_admin.initialise_cli import InitialiseShell
 
 DIR_PATH = os.path.dirname(os.path.abspath(__file__))
 
 
-class InitialiseShell(cmd.Cmd):
-    intro = """--- Menu ---
-(1) Clear database contents
-(2) Clear database schema
-(3) Create Pepys schema
-(4) Import Reference data
-(5) Import Metadata
-(6) Import Sample Measurements
-(0) Exit
-"""
-    prompt = "(initialise) "
-
-    def __init__(self, data_store, parent_shell, csv_path):
-        super(InitialiseShell, self).__init__()
-        self.data_store = data_store
-        self.csv_path = csv_path
-        self.aliases = {
-            "0": self.do_cancel,
-            "1": self.do_cleardb_contents,
-            "2": self.do_cleardb_schema,
-            "3": self.do_create_pepys_schema,
-            "4": self.do_import_reference_data,
-            "5": self.do_import_metadata,
-            "6": self.do_import_sample_measurements,
-        }
-
-        if parent_shell:
-            self.prompt = parent_shell.prompt.strip() + "/" + self.prompt
-
-    def do_cleardb_contents(self):
-        self.data_store.clear_db_contents()
-        print("Cleared database contents")
-
-    def do_cleardb_schema(self):
-        self.data_store.clear_db_schema()
-        print("Cleared database schema")
-
-    def do_create_pepys_schema(self):
-        self.data_store.initialise()
-        print("Initialised database")
-
-    def do_import_reference_data(self):
-        with self.data_store.session_scope():
-            self.data_store.populate_reference(self.csv_path)
-        print("Reference data imported")
-
-    def do_import_metadata(self):
-        with self.data_store.session_scope():
-            self.data_store.populate_metadata(self.csv_path)
-        print("Metadata imported")
-
-    def do_import_sample_measurements(self):
-        with self.data_store.session_scope():
-            self.data_store.populate_measurement(self.csv_path)
-        print("Sample measurements imported")
-
-    @staticmethod
-    def do_cancel():
-        return True
-
-    do_EOF = do_cancel
-
-    def default(self, line):
-        cmd, arg, line = self.parseline(line)
-        if cmd in self.aliases:
-            self.aliases[cmd](arg)
-            if cmd == "0":
-                self.do_cancel()
-                return True
-        else:
-            print("*** Unknown syntax: %s" % line)
-
-    def postcmd(self, stop, line):
-        if line != "0":
-            print(self.intro)
-        return cmd.Cmd.postcmd(self, stop, line)
-
-
 class AdminShell(cmd.Cmd):
     intro = """--- Menu ---
 (1) Export
 (2) Initialise/Clear
 (3) Status
+(9) Export All

```

Comment:
@BarisSari - option 9 was deliberately hidden.  `Export All` isn't a usage of Pepys that we encourage, but it's something that's useful for testing.

Reply:
Okay. It's deleted:
https://github.com/debrief/pepys-import/pull/253/commits/31d0eaa1b38daba4d6920a7557fcd0e705922cce